client/me/purchases/cancel-purchase/index.jsx

```diff
@@ -127,14 +127,20 @@ class CancelPurchase extends React.Component {
 		const renewalDate = this.props.moment( renewDate ).format( 'LL' );
 
 		if ( isDomainRegistration( purchase ) ) {
-			return this.props.translate( 'Domain will be removed on %(renewalDate)s', {
-				args: { renewalDate },
-			} );
+			return this.props.translate(
+				'When you click to confirm, the domain will be removed on %(renewalDate)s',
```

Comment:
I agree - "After" sounds a lot better.

Also, while nitpicking, it's generally a good idea to avoid using words like "click" in UI text... Maybe it would be best to just have "After you cancel, the subscription will be removed on [date]"?